blockchain/config.go

```diff
@@ -37,6 +37,7 @@ type (
 		ID                         uint32           `yaml:"id"`
 		EVMNetworkID               uint32           `yaml:"evmNetworkID"`
 		Address                    string           `yaml:"address"`
+		ReplayDeployerWhitelist    []string         `yaml:"replayDeployerWhitelist"`
```

Comment:
what if the field is added to genesis?

Reply:
genesis.Blockchain is also an option, but looking at the fields I feel it is more appropriate to add here?

Reply:
are you sure?